README.md

```diff
@@ -69,3 +69,21 @@ Provider app should implement a short-lived RPC service which implements the `Pr
   - takes no arguments
   - returns a score which indicates how efficient the machine is for this type of tasks
   - shouldn't take much time (preferably less than a minute for medium range machines)
+
+# Running generator
+
+After updating the .proto files you need to re-generate the python code and commit the changes together.
+
+## Install requirements
+To install the requirements for the generator you run:
+
+```
+python python/setup.py develop
```

Comment:
I'd put Python specific instructions inside `python` directory's readme and leave some generic description at the top level.

Reply:
I started changing it but not sure if i agree.

These requirements are needed when changing the `proto/*.proto` files and to run `make` in the main repository. So i think they are global guidelines for using this repository.

Reply:
Fair enough. It'd make sense to put these commands into the `make python` command tho.
This way you just run language-agnostic `make` and are done.

Reply:
ok, updated the doc to use `make python` and keep it language specific, this is the final goal :)

Reply:
ok, changed the doc to use `make python` to keep it language specific, this is the final goal :)